daemon/daemon_unix.go

```diff
@@ -1,5 +1,5 @@
-//go:build darwin
-// +build darwin
+//go:build darwin || openbsd
+// +build darwin openbsd
```

Comment:
how about adding freebsd and netbsd?

Reply:
:man_facepalming: How could i ever ?!?!?! :smile: adding the rest of the family

Reply:
What do you think if we make this our fall back for unix systems and simply exclude linux/windows on top ?

There are a lot more unix systems that will work with this one rather with the linux.

Reply:
I like the `!linux && !windows`